prime-router/src/main/kotlin/cli/tests/RemovePIITest.kt

```diff
@@ -56,40 +59,48 @@ class RemovePIITest : CoolTest() {
 
         if (!testIdsRemoved(inputBundle, outputContent)) {
             ugly("Not all IDs removed. Test failed.")
+            Path(outputFilePath).deleteIfExists()
             return false
         }
 
         if (!testPatientPIIRemoved(inputBundle, outputBundle)) {
             ugly("Not all patient PII removed. Test failed.")
+            Path(outputFilePath).deleteIfExists()
             return false
         }
 
         if (!testOrganizationPIIRemoved(inputBundle, outputBundle)) {
             ugly("Not all organization PII removed. Test failed.")
+            Path(outputFilePath).deleteIfExists()
             return false
         }
 
         if (!testPractitionerPIIRemoved(inputBundle, outputBundle)) {
             ugly("Not all practitioner PII removed. Test failed.")
+            Path(outputFilePath).deleteIfExists()
             return false
         }
 
         if (!testServiceRequestPIIRemoved(inputBundle, outputBundle)) {
             ugly("Not all service request PII removed. Test failed.")
+            Path(outputFilePath).deleteIfExists()
             return false
         }
 
         if (!testObservationPIIRemoved(inputBundle, outputBundle)) {
             ugly("Not all observation PII removed. Test failed.")
+            Path(outputFilePath).deleteIfExists()
```

Comment:
This implementation is risky because for new use cases you need to remember to have this line